workflow_scripts/run_mlagility.py

```diff
@@ -0,0 +1,38 @@
+import config
+import os.path as osp
+from pathlib import Path
+import shutil
+import subprocess
+import sys
+
+
+base_name = "-op18-base.onnx"
+cwd_path = Path.cwd()
+mlagility_root = "mlagility/models"
+mlagility_models_dir = "models/mlagility"
+cache_converted_dir = ".cache"
+ZOO_OPSET_VERSION = "18"
+
+errors = 0
+
+for script_path, model_name, model_zoo_path in config.models_info:
+    try:
+        print(f"----------------Checking {model_zoo_path}----------------")
+        final_model_path = osp.join(mlagility_models_dir, model_zoo_path.replace(".onnx", "-" + ZOO_OPSET_VERSION + ".onnx"))
+        subprocess.run(["benchit", osp.join(mlagility_root, script_path), "--cache-dir", cache_converted_dir, "--onnx-opset", ZOO_OPSET_VERSION],
```

Comment:
MLAgility introduces `--export-only` flag [here](https://github.com/groq/mlagility/blob/c683577f3284ec9665d8916494e0cef1536ac47c/docs/tools_user_guide.md?plain=1#L513)
I would recommend using that to generate the onnx file.

Reply:
Good point. I thought we can run more here (for instance, fp16 conversion or other optimization), but it seems export is sufficient here, because we just want to ensure the provided model is consistent with what benchit created. I have updated it with `export-only`